README.MD

```diff
@@ -111,9 +101,11 @@ let deployParams = {
   gas: '7500000',
   gasPrice: '0x3B9ACA00'
 };
+let organizationContractInstance;
 let organizationAddress;
-organizationHelperObject.setup(organizationConfig, deployParams).then(function(){
-    organizationAddress = organizationHelperObject.address;
+Organization.setup(originWeb3, orgConfig).then(function(instance){
```

Comment:
I am not sure this README should repeat documentation from other repositories, as it goes easily out-of-sync. Instead, maybe it could link to the documentation of the other repo? But even that link could break...

What do you think?

/cc @sarvesh-ost @benjaminbollen

Reply:
I think it's good to link documentation. 
Once we move to stable releases, link breakage will only happen on releases.  Which can be updated in every new release of the branded token.

Reply:
Old releases' documentation would actually remain 👍